benchmark.sh

```diff
@@ -40,6 +40,12 @@ fi
 
 RES_DIR=$1; shift
 
+# To simplify path handling, require the results dir to be absolute.
+if [ "$(realpath "${RES_DIR}")" != "${RES_DIR}" ]; then
```

Comment:
Do the inner `"`s have any effect here?

Reply:
I think it allows paths to have spaces, but why would we? Shall we remove them?

Reply:
Yes, I would tend to remove them, because I'm very unsure what they do when nested. Out of curiosity, what does `shellcheck` say? It tends to be pretty good at these things. If it's happy with it, then I'm probably happy with it too!

Reply:
OK, I did not know that inner quotes worked like that -- interesting!